WordPress/Classes/ViewRelated/Post/EditImageDetailsViewController.m

```diff
@@ -525,9 +525,9 @@ - (void)showAlignmentSelector
 
     PostSettingsSelectionViewController *vc = [[PostSettingsSelectionViewController alloc] initWithDictionary:dict];
     __weak PostSettingsSelectionViewController *weakVc = vc;
-    vc.onItemSelected = ^(NSString *status) {
+    vc.onItemSelected = ^(NSObject *status) {
         // do interesting work here... like updating the value of image meta.
-        self.imageDetails.align = status;
+        self.imageDetails.align = (NSString *)status;
```

Comment:
Me, being annoying (again): Is status *always* gonna be a String?. If so, (maybe) we should keep the right type in the block callback. And if not, (maybe) we should run a safety check, just to make sure we got a String.